lib/node_runner_message_checker_test.go

```diff
@@ -0,0 +1,118 @@
+package sebak
+
+import (
+	"testing"
+
+	"github.com/stellar/go/keypair"
+	"github.com/stretchr/testify/assert"
+
+	"boscoin.io/sebak/lib/common"
+	"boscoin.io/sebak/lib/error"
+	"boscoin.io/sebak/lib/network"
+	"boscoin.io/sebak/lib/node"
+	"boscoin.io/sebak/lib/storage"
+)
+
+func MakeNodeRunner() (*NodeRunner, *sebaknode.LocalNode) {
```

Comment:
I think we have about a dozen of those functions at this point... This kind of utility should find its way to `lib/test.go`